addons/controllers/clusterbootstrap_controller.go

```diff
@@ -575,6 +574,21 @@ func (r *ClusterBootstrapReconciler) mergeClusterBootstrapPackagesWithTemplate(
 			updatedClusterBootstrap.Spec.AdditionalPackages = append(updatedClusterBootstrap.Spec.AdditionalPackages, newPkg)
 			packages = append(packages, newPkg)
 		}
+		additionalPackageTemplateMap[packageRefName] = templatePkg
+	}
+
+	// There might be packages in clusterbootstrap but not in clusterboostraptemplate.
+	// The below loop tries to find those packages and updates the RefName if it is found in tkr's spec.bootstrapPackages.
+	// If its not there in tkr's spec.bootstrappackages then user has to update the version of it in clusterbootstrap.
```

Comment:
If its not in tkr's ```bootstrappackages```, does it work if the user just update the version in ```clusterbootstrap```? Should we provide instructions on how? Wouldn't it immutable?

Reply:
I have added a testcase to update the version in clusterbootstrap and ensured it can work. I am not sure there is anything special to provide in documentation. It's just a matter of updating clusterbootstrap.spec.AdditionalPackages[index].RefName to updated version